Python/Advanced_Function_Lambda/README.md

```diff
@@ -0,0 +1 @@
+[![Advanced Python Functions: Lambda (V)](Lambda-574-Thumbnail.png)](https://drive.google.com/file/d/10jVjCcxN67BbPKQPOxnIyuyTh_D7LT8Y/view?usp=sharing "Lambda Functions (Video)")
```

Comment:
@srini047 amazing work on the video! just make your md file more presentable: 
- Give a proper heading 
- Write down the subtopics covered in the video

Reply:
I have changed the READM.md file as per the suggestions above. @apeksha235

Reply:
Amazing! @adityabisoi please merge this